mappings/net/minecraft/command/arguement/ResourceOrTagLocationArgument.mapping

```diff
@@ -0,0 +1,47 @@
+CLASS net/minecraft/unmapped/C_kmrsoqit net/minecraft/command/arguement/ResourceOrTagLocationArgument
```

Comment:
Typo of `argument` in the package name, the class name should be adapted to Identifier instead of ResourceLocation